frontend/stores/list-watch.js

```diff
@@ -0,0 +1,41 @@
+import { observable, action } from 'mobx'
+import { createContext } from 'react'
+import axios from 'axios'
+
+class ListWatchStore {
+  @observable watches = []
+
+  @action
+  async fetchWatches(userId) {
```

Comment:
```suggestion
  async fetch(userId) {
```
`fetch()` là đủ tường minh rồi, đây là watch list, em còn fetch cái gì được nữa?